lib/api/src/c_api/mod.rs

```diff
@@ -33,6 +33,9 @@ mod lib {
     }
 }
 
+/// A constant that indicates to users that the WAMR (interpreter) backend is in use. 
+pub const WASMER_WAMR: bool = true;
```

Comment:
We should not use this. The engine should return this @xdoardo

Reply:
Here: https://github.com/wasmerio/wasmer/pull/4515/files#diff-5d10c39c36980e1ba420d516d4bd3d7202ddf63c08a3366940d73dcd3f522868R30